include/core/cfdp_core/pdu_directive.hpp

```diff
@@ -0,0 +1,33 @@
+#include "cfdp_core/pdu_enums.hpp"
+#include "cfdp_core/pdu_interface.hpp"
+#include <cstdint>
+#include <span>
+namespace cfdp::pdu::directive
+{
+
+class KeepAlivePdu : PduInterface
+{
+  public:
+    KeepAlivePdu(uint32_t progress);
```

Comment:
I would put member initializer constructors in .hpp if they don't have any additional logic

Reply:
easier to read